src/main/java/de/tum/cit/aet/artemis/communication/service/FaqService.java

```diff
@@ -0,0 +1,52 @@
+package de.tum.cit.aet.artemis.communication.service;
+
+import static de.tum.cit.aet.artemis.core.config.Constants.PROFILE_CORE;
+
+import java.util.Optional;
+import java.util.Set;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import de.tum.cit.aet.artemis.communication.domain.Faq;
+import de.tum.cit.aet.artemis.communication.repository.FaqRepository;
+
+/**
+ * REST service for managing Faqs.
+ */
+@Profile(PROFILE_CORE)
+@Service
+public class FaqService {
```

Comment:
If your service just "directs" the calls to the repository, you can omit calling the `FaqService` and directly call the `FaqRepository` in the rest controller. But that's rather a personal opinion.

Reply:
I followed coderabits remarks about seperation of concerns here. It proposed me to not grant direct access for the ressource but to use the Service instead